Zone5/ThirdPartyConnections/ThirdPartyResponse.swift

```diff
@@ -8,7 +8,8 @@
 
 import Foundation
 
-/// Response from ThirdPartyConnectionsView.setThirdPartyToken and removeThirdPartyToken
-public struct ThirdPartyResponse: Codable {
-	public let success: Bool
+/// Response from ThirdPartyConnectionsView.hasThirdPartyToken
+struct ThirdPartyResponse: Codable {
+    let type: String
+	let enabled: Bool
```

Comment:
It looks like your spacing is different than what was used for this file and `ThirdPartyConnectionsView`.

Reply:
Fixed